client/src/components/PopoverField/PopoverField.js

```diff
@@ -103,6 +115,7 @@ PopoverField.defaultProps = {
   data: {},
   className: '',
   popoverClassName: '',
+  buttonSizeClassName: '',
```

Comment:
default `btn--icon-xl` here :)

Or if we go with just `buttonSize` prop, this could be just `xl`

Reply:
Sounds good, will update